pkg/tcpip/stack/nic.go

```diff
@@ -663,10 +663,10 @@ func (n *NIC) DeliverTransportPacket(r *Route, protocol tcpip.TransportProtocolN
 	}
 
 	id := TransportEndpointID{dstPort, r.LocalAddress, srcPort, r.RemoteAddress}
-	if n.demux.deliverPacket(r, protocol, netHeader, vv, id) {
+	if n.demux.deliverPacket(r, protocol, netHeader, vv, id, true) {
```

Comment:
We often insert a comment on literal arguments. It helps with the readability as it isn't clear what the literal means in this context.

Example:
```
n.demux.deliverPacket(r, protocol, netHeader, vv, id, true /* first */)

Same below.
```

Reply:
Consider doing this with `getLine` in `hostinet/stack.go` as well.

Reply:
Done.